packages/alfa-dom/test/node/query/element-descendants.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import { test } from "@siteimprove/alfa-test";
+import { h } from "../../../h";
+
+import { Query } from "../../../src";
+
+const { getElementDescendants } = Query;
+
+test("#getElementDescendants() returns descendants non-inclusively", (t) => {
```

Comment:
**Nitpick:** This technically doesn't test non-inclusivity since `doc` is not an `Element` and would be filtered away anyway.

Reply:
Ah, yes, good catch